usefulUtilities/hiFiCalendar/clientScripts/meetingRoomSetup_app.js

```diff
@@ -0,0 +1,158 @@
+//  meetingRoomSetup_app.js
+//
+//  Created by Mark Brosche on 4-2-2019
+//  Handed off to Milad Nazeri on 5-15-2019
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+    var CONFIG = Script.require("../calendarConfig.json?" + Date.now());
```

Comment:
you got it. I think we do need to be more cognizant of this in general.  I'll get all of them out in here though.

Reply:
wait - @huffman we do this on purpose because we want to be able to change the config without requiring people to reload content.

@MiladNazeri please don't remove this unless there's some reason i haven't considered.

Reply:
Ahh, sorry Zach, that is a good point re: configs